analyzer/src/main/resources/UserAgents/AppleTypes.yaml

```diff
@@ -114,8 +114,6 @@ config:
       "iPhone10,6" : "Phone"
       "iPhone10C6" : "Phone"
       "iPhone X" : "Phone"
-      "iPhone X" : "Phone"
```

Comment:
This is generated code.
Because eveything gets located into a hashmap these duplicates have almost no impact.
To fix this the code generator must be changed.